internal/ui/termstatus/status_test.go

```diff
@@ -1,6 +1,7 @@
 package termstatus
 
 import (
+	"reflect"
```

Comment:
Goes with the suggestion below:

```suggestion
	"fmt"
```